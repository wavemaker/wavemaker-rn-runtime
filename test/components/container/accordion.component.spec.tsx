import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmAccordion from '@wavemaker/app-rn-runtime/components/container/accordion/accordion.component';

describe('Test Accordion component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmAccordion name="test_Accordion"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});