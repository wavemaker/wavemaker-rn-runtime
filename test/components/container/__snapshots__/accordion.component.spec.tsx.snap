// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Accordion component Check validity of sample component 1`] = `
<View
  style={
    {
      "backgroundColor": "transparent",
      "borderColor": "#eeeeee",
      "borderRadius": 6,
      "borderStyle": "solid",
      "borderWidth": 0,
      "width": "100%",
    }
  }
/>
`;
